packages/indexer-client/src/IndexerClient.ts

```diff
@@ -7,36 +7,36 @@ import { toBigDecimal } from '@vertex-protocol/utils';
 import { IndexerBaseClient } from './IndexerBaseClient';
 import {
   BaseIndexerPaginatedEvent,
-  GetIndexerPaginatedInterestFundingPaymentsResponse,
-  GetIndexerPaginatedOrdersParams,
-  GetIndexerPaginatedOrdersResponse,
-  GetIndexerPaginatedRewardsParams,
-  GetIndexerPaginatedRewardsResponse,
-  GetIndexerSubaccountCollateralEventsParams,
-  GetIndexerSubaccountCollateralEventsResponse,
-  GetIndexerSubaccountInterestFundingPaymentsParams,
-  GetIndexerSubaccountLiquidationEventsParams,
-  GetIndexerSubaccountLiquidationEventsResponse,
-  GetIndexerSubaccountLpEventsParams,
-  GetIndexerSubaccountLpEventsResponse,
-  GetIndexerSubaccountMatchEventParams,
-  GetIndexerSubaccountMatchEventsResponse,
-  GetIndexerSubaccountSettlementEventsParams,
-  GetIndexerSubaccountSettlementEventsResponse,
   IndexerCollateralEvent,
   IndexerEventPerpStateSnapshot,
   IndexerEventSpotStateSnapshot,
   IndexerEventWithTx,
   IndexerLiquidationEvent,
   IndexerLpEvent,
+  IndexerQueryPaginatedInterestFundingPaymentsResponse,
+  IndexerQueryPaginatedOrdersParams,
+  IndexerQueryPaginatedOrdersResponse,
+  IndexerQueryPaginatedRewardsParams,
+  IndexerQueryPaginatedRewardsResponse,
+  IndexerQuerySubaccountCollateralEventsParams,
+  IndexerQuerySubaccountCollateralEventsResponse,
+  IndexerQuerySubaccountInterestFundingPaymentsParams,
+  IndexerQuerySubaccountLiquidationEventsParams,
+  IndexerQuerySubaccountLiquidationEventsResponse,
+  IndexerQuerySubaccountLpEventsParams,
+  IndexerQuerySubaccountLpEventsResponse,
+  IndexerQuerySubaccountMatchEventParams,
+  IndexerQuerySubaccountMatchEventsResponse,
+  IndexerQuerySubaccountSettlementEventsParams,
+  IndexerQuerySubaccountSettlementEventsResponse,
   IndexerSettlementEvent,
   PaginatedIndexerEventsResponse,
 } from './types';
 
 export class IndexerClient extends IndexerBaseClient {
   async getPaginatedSubaccountMatchEvents(
-    params: GetIndexerSubaccountMatchEventParams,
-  ): Promise<GetIndexerSubaccountMatchEventsResponse> {
+    params: IndexerQuerySubaccountMatchEventParams,
```

Comment:
it seems some params / response include `paginated` e.g: `IndexerQueryPaginatedOrdersParams` and some don't, think we should standardize it.

Reply:
Hm these are paginated queries with pagination `meta`, but the queries on `IndexerBaseClient` don't really have pagination built in